src/input.service.ts

```diff
@@ -278,21 +336,23 @@ export class InputService {
   private subscriptions: Subscription[] = [];
   private pollRaf: number = null;
 
-  constructor(private focus: FocusService) {}
+  constructor(private focus: FocusService) { }
 
   /**
    * Bootstrap attaches event listeners from the service to the DOM and sets
    * up the focuser rooted in the target element.
    */
-  public bootstrap(root: HTMLElement = document.body) {
+  public bootstrap(handleTab : (direction : Direction) => any = (direction : Direction) => { direction }, root: HTMLElement = document.body) {
+    this.handleTab = handleTab;
```

Comment:
I'm not sure it's the role of the FocusService to provide these kind of global handlers. Consumers can listen to the gamepad themselves if they need a global handler; if they want to handle input given within a specific element, they can attach standard arc listeners.